src/components/ScreenWrapper.tsx

```diff
@@ -89,7 +89,7 @@ type ScreenWrapperProps = {
      *
      * This is required because transitionEnd event doesn't trigger in the testing environment.
      */
-    navigation?: StackNavigationProp<RootStackParamList>;
+    navigation?: StackNavigationProp<RootStackParamList> | StackNavigationProp<CentralPaneNavigatorParamList>;
```

Comment:
```suggestion
    navigation?: StackNavigationProp<RootStackParamList | CentralPaneNavigatorParamList>;
```

Does it work?

Reply:
@fabioh8010 this way it gives the error on the Report screen
![image](https://github.com/Expensify/App/assets/23176449/14a18f54-2e98-4ed0-bf80-00587f0e11c0)